frontend/src/components/nodes/__test__/__snapshots__/NodesEpoch.test.tsx.snap

```diff
@@ -67,16 +67,7 @@ exports[`<NodesEpoch /> renders 1`] = `
         >
           <svg
             className="rc-progress-circle progress-bar-content"
-            label={
```

Comment:
TODO: find out what happened here

Reply:
Yep, we probably don't need to pass attributes "type" and "label" to svg node :)